app/assets/sass/application.scss

```diff
@@ -203,6 +211,16 @@ table.override-gov-uk-table tr{
   line-height: 40px;
 }
 
+.table-first-visible-child {
```

Comment:
There is CSS to select the first visible child, but this is overriding an MoJ sortable table style which is for the first child. I chose to make it more explicit.